notesapi/v1/views.py

```diff
@@ -1,24 +1,29 @@
 import json
 import logging
+from functools import lru_cache
```

Comment:
Imported but not used! Sorry for clinging.

Reply:
Missed that. I've deleted it. Thanks.